payments/mobile-pay/other-features.md

```diff
@@ -146,7 +146,7 @@ The response will be the `payment` resource with its `state` set to `Aborted`.
 
 {% include callback-reference.md %}
 
-{% include subsite.md %}
+{% include subsite.md mobilepay=true %}
```

Comment:
Instead of having one boolean parameter per payment instrument, perhaps `subsite` can take `payment-instrument` as a string instead?